src/lib/services-manager/clients/services_manager.rb

```diff
@@ -0,0 +1,399 @@
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+
+Yast.import "ServicesManager"
+Yast.import "UI"
+Yast.import "Wizard"
+Yast.import "Service"
+Yast.import "Label"
+Yast.import "Popup"
+Yast.import "Report"
+Yast.import "Message"
+Yast.import "Mode"
+Yast.import "CommandLine"
+Yast.import "SystemdService"
+Yast.import "PackageSystem"
+
+module Y2ServicesManager
+  module Clients
+    class ServicesManager < Yast::Client
+      include Yast::Logger
+
+      module Id
+        SERVICES_TABLE = :services_table
+        TOGGLE_RUNNING = :start_stop
+        TOGGLE_ENABLED = :enable_disable
+        DEFAULT_TARGET = :default_target
+        SHOW_DETAILS   = :show_details
+        SHOW_LOGS      = :show_logs
+      end
+
+      # Constructor
+      #
+      # Journal package (yast2-journal) is not an strong dependency (only suggested).
+      # Here the journal is tried to be loaded, avoiding to fail when the package is
+      # not installed (see {#load_journal}).
+      def initialize
+        load_journal
+      end
+
+      def run
+        textdomain 'services-manager'
+
+        cmdline = {
+          "id"         => "services-manager",
+          # translators: command line help text for services-manager module
+          "help"       => _(
+                            "Systemd target and services configuration module.\n" +
+                            "Use systemctl for commandline services configuration."
+                            ),
+          "guihandler" => fun_ref(method(:gui_handler), "boolean ()")
+        }
+
+        CommandLine.Run(cmdline)
+      end
+
+      def gui_handler
+        Wizard.CreateDialog
+        success = false
+        while true
```

Comment:
`loop`?

Reply:
I have not touched nothing here, is only the existing code. Do you think we should modify it?

Reply:
just NP, so it is up to you. But if we want as part of touching this code also use rubocop, I am sure rubocop say you same thing :)

Reply:
You are right. But there are several things to adapt for rubocop. To avoid possible changes in behavior, I would leave it as it is, and then we can improve the code to pass rubocop in a separated card.